### ISSUE TITLE
Merge/Inheritance of Serializers

### ISSUE THREAD

**Issue by Fleick** [Netflix/fast_jsonapi#107]
Is there any possibility to use attribute definitions of one Serializer in another?
Like i have:

class SimpleDataSerializer
  attributes :a, :b, :c
end

class FullDataSerializer
  attributes :a, :b, :c, :d, :e
end

#Scopes
scope :simple
...
scope full, -> do
  merge(x.simple).select(additionalData)
end

for many Objects. so it would be nice to include simple in full Serializer and avoid double writing work. Especially to avoid dopple work when attributes in simple change.

**Comment by shishirmk** [comment]
@Fleick You could create a separate module/concern and put the shared attribute definitions in the included do block of such a module.

```
  module SharedAttributes
    extend ActiveSupport::Concern

    included do
      attributes :a, :b, :c
    end
```

**Comment by shishirmk** [comment]
Serializer inheritance should work as expected now. This is released in version 1.2 of the gem
